include/materials/norsand.h

```diff
@@ -69,12 +69,12 @@ class NorSand : public Material<Tdim> {
   void compute_plastic_tensor(const Vector6d& stress,
                               mpm::dense_map* state_vars);
 
-  //! Compute stress invariants (p, q, j2, j3, lode_angle and M_theta)
+  //! Compute stress invariants (p, q, lode_angle and M_theta)
   //! \param[in] stress Stress
   //! \param[in] state_vars History-dependent state variables
-  //! \retval vector of size six, containing p, q, j2, j3, lode_angle and
+  //! \retval vector of size six, containing p, q, lode_angle and
   //! M_theta
-  Eigen::Matrix<double, 6, 1> compute_stress_invariants(
+  Eigen::Matrix<double, 4, 1> compute_stress_invariants(
```

Comment:
If you need 4 different return values, could you please pass them as a pointer instead of returning an Eigen Matrix.

Reply:
Can you comment on the current design? I still don't pass the test function which means something is wrong

Reply:
I have successfully pass the test functions so it's OK now. Problem with `const`